### DISCUSSION TITLE
Has anyone installed Panopticon on Inmotion Hosting?

### DISCUSSION THREAD

**Discussion by phildavies** [akeeba/panopticon#666] [Q&A (Community Support)] [answered]
Hi I'm trying to set up Panopticon on Inmotion hosting but am getting stuck with setting up the Cron Job.  I contacted Inmotion Hosting support to get the /path/to/php with the path to the PHP 8.2.16 CLI executable and they gave me this path /opt/cpanel/ea-php82/root/usr/bin/php .  So in the Cron Job command line I have /opt/cpanel/ea-php82/root/usr/bin/php/home/parawa6/mywebsitedomain.com/cli/panopticon.php task:run --loop >/dev/null 2>&1  When I try that nothing happens so I followed the instruction to remove the 2>&1 from the CRON command line and then I get the error email message that tells me /bin/bash: /opt/cpanel/ea-php82/root/usr/bin/php/home/parawa6/mywebsitedomain.com/cli/panopticon.php: Not a directory.  Has anyone successfully set up Panopticon on Inmotion hosting and can tell me where I've gone wrong?  As Inmotion Hosting support weren't sure and had to contact a different department to give me the /path/to/php and so it could be that isn't correct.  Thanks for any help anyone can give.

**Comment by nikosdion** [comment, marked as answer]
You forgot the all important **space** between the PHP executable path `/opt/cpanel/ea-php82/root/usr/bin/php` and the path to Panopticon's CLI PHP file `/home/parawa6/mywebsitedomain.com/cli/panopticon.php`.

**Reply by phildavies** [reply]
Thank you nikosdion yes that was the mistake I'd made :)
